### PR TITLE
Typescript3 update

### PR DESCRIPTION
**Pull request by warrenv**
References https://github.com/poetapp/random/issues/116

### Description of Changes

Upgrade typescript package to version 3.

### PR Review Checklist

* [ ] **Do the changes match the product spec?**
* [ ] **Are there changes in this PR that weren't originally in the product spec?**
* [ ] **Is a feature toggle introduced for this PR? Is it disabled by default?**
* [ ] **Have you tested the changes?**
* [ ] **Is QE review required? If so, has this been reviewed by QE?** (For all new features, review by QE is required. Minor changes which already have passing functional tests may not need a new QE review -- when in doubt, loop in a QE reviewer).
* [ ] **Are there code changes that require new unit tests? If so, are those tests included?** (A separate PR for tests is not appropriate).
* [ ] **Are documentation updates required? If so, does this PR include it?** (Separate documentation PR is generally not appropriate).
* [ ] **Are the added/edited files named appropriately and located correctly in the project directory structure?**

### Warning

Merging to `master` automatically triggers a deploy in production.

### CODE REVIEW

File: src/components/organisms/Footer.tsx [hunk 1]
```diff
@@ -4,7 +4,9 @@ import { Images } from '../../images/Images'
 
 import './Footer.scss'
 
-export class Footer extends React.Component<undefined, undefined> {
+interface FooterProps {}
```
Comment (lautarodragan) [inline src/components/organisms/Footer.tsx:7]: Is this change needed for the version upgrade?

Comment (warrenv) [reply src/components/organisms/Footer.tsx:7]: yes, the build fails without it.

**Review by lautarodragan — DISMISSED** [review]
